### PR TITLE
More fde strings

### PR DESCRIPTION
**Pull request by dbungert**
@mwhudson is a reason available in the DISABLED case?

### CODE REVIEW

**Comment by mwhudson** [comment]
> @mwhudson is a reason available in the DISABLED case?

(some time passes, sorry)

I think not, and it seems from looking at snapd that the DISABLED case is a bit of an odd one that we are unlikely to run into in a "real" environment:

```
	if dangerous && osutil.FileExists(filepath.Join(boot.InitramfsUbuntuSeedDir, ".force-unencrypted")) {
```

Looks like it's basically there to support a hackish unencrypted install of a dangerous model, presumably for development purposes only?

**Comment by mwhudson** [comment]
what is the status of this branch btw?

**Comment by dbungert** [comment]
I think it's ready, albeit not tested terribly well as I'm not all that up to speed on this work.

**Comment by dbungert** [comment]
Mostly I think it was the hope that there was more info for the disabled case, and if there isn't we should move on.

File: subiquity/ui/views/filesystem/guided.py [hunk 1]
```diff
@@ -129,41 +129,43 @@ class TPMChoice:
...
-            help=_("The entire disk will be encrypted and protected by the "
-                   "TPM. If this option is deselected, the disk will be "
-                   "unencrypted and without any protection.")),
+            help=help_texts[StorageEncryptionSupport.AVAILABLE]),
         StorageSafety.PREFER_UNENCRYPTED: TPMChoice(
             enabled=True, default=False,
-            help=_("The model being installed does not prefer but allows TPM "
-                   "backed full-disk encryption")),
-            },
+            help=help_texts[StorageEncryptionSupport.AVAILABLE]),
```
Comment (mwhudson) [inline subiquity/ui/views/filesystem/guided.py:161]: I'm not sure using the same text in all AVAILABLE cases makes sense, in particular for the ENCRYPTED/AVAILABLE case the text will say "If this option is deselected" but that's not going to be possible. Maybe this is OK, but it feels a bit strange to me.

**Review by mwhudson — APPROVED** [review]
This is an improvement but I do have one quibble (which I am happy for you to ignore)

File: subiquity/ui/views/filesystem/guided.py (hunk 1 above)
Comment (dbungert) [reply subiquity/ui/views/filesystem/guided.py]: That sounds like a common case and talking about an impossible deselection would look awkward.  Happily fixed.

**Pull request merged**